Exception occurred

Exception: <%= @exception.class.name %>
Message:   <%= @exception.message %>

Backtrace:
<%= @exception.backtrace.join("\n") %>
